server.go

```diff
@@ -504,12 +504,19 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB, cc *chainControl,
 	}
 	s.currentNodeAnn = nodeAnn
 
+	// The router will get access to the payment ID sequencer, such that it
```

Comment:
Even if it's not used by the switch directly, it is exposed so that users of the `htlcswitch` package properly adhere to the semantics required by `SendHTLC`.